assets/css/sensei-course-theme/quiz-graded.scss

```diff
@@ -0,0 +1,56 @@
+.sensei-course-theme-quiz-graded-notice {
+	padding-bottom: 36px;
+
+	&__title {
+		font-size: 30px;
+		font-weight: 700;
+		line-height: 1.2;
+		margin: 0;
+		margin-bottom: 6px;
+	}
+
+	&__text {
+		font-size: 16px;
+		line-height: 1.75;
+		margin: 0;
+		margin-bottom: 42px;
+	}
+
+	&__actions {
+		display: flex;
+		flex-direction: row;
+		justify-content: flex-start;
+		align-items: center;
+		gap: 24px;
+	}
+
+	&__reset-quiz-form {
+		margin-bottom: 0;
+	}
+
+	&__reset-quiz-button {
```

Comment:
I think these 2 buttons could share the same class.

Also, they could use `sensei-course-theme__button is-link`. The only place using `is-link` currently is the `templates/course-theme/lesson-quiz-notice.php` (which is not with the correct design yet):

<img width="920" alt="Screen Shot 2021-12-27 at 15 08 42" src="https://user-images.githubusercontent.com/876340/147496868-81fa0314-90e2-4ab1-8a2c-5e4a4d33c6cb.png">

We could tweak the `is-link` (`assets/css/sensei-course-theme/buttons.scss`) to something like that:

```
	&.is-link {
		font-weight: 600;
		text-decoration: underline;
	}
```

WDYT?

Reply:
Agree. That makes it much simpler. Updated. a2c8755a29af5bfcc5537156cc13931307312514